.github/workflows/release.yaml

```diff
@@ -1,25 +1,96 @@
-name: release
+name: Create Release
 
 on:
   push:
-    # Sequence of patterns matched against refs/tags
-    tags:
-    - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
-
+    branches:
+      - main
+    paths:
+      - 'CHANGELOG/*.md'
+  
 permissions:
-  contents: write # Allow to create a release.
+  contents: write # Allow to push a tag.
```

Comment:
Nit: Probably should mention everything we do or nothing :) (tag, branch, release)